mabl snippets/waitUntilNotPresent.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * TITLE: Wait until the provided element is no longer present up to 15 seconds
+ *        testing every 100 ms
+ *
+ * @mablParam: elementSelector - A CSS selector for the element
+ *
+ * @mablReturn: 'true' if the element disappeared within the timeout,
+ *              'false' otherwise
+ *
+ */
+async function mablJavaScriptStep(mablInputs, callback) {
```

Comment:
Is IE compatibility important here? I don't think IE supports async.

Reply:
Spot on. I updated the documentation above accordingly.